### PR TITLE
The graph page crashes if no selection was done on the search modal

### PR DESCRIPTION
**Pull request by ds161833**
Fixed null exception

### CODE REVIEW

**Review by GD-Dheer — APPROVED** [review]
LGTM. Nice catch

**Review by Mehrdadmaskull — APPROVED** [review]
LGTM

File: Client/src/Components/Search/SearchView.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ export default function SearchView(props: IProps) {
   const [foundDatasets, setFoundDatasets] = useState<IDatasetModel[]>([])
 
   // array of selected ids
-  const [selection, setSelection] = useState<SelectionChangeParams>(null)
+  const [selection, setSelection] = useState<SelectionChangeParams>({ rowIds: [] })
```
Comment (ratulkai93) [inline Client/src/Components/Search/SearchView.tsx:21]: how does the rowId's work here?

**Review by ratulkai93 — APPROVED** [review]
Looks good over all

File: Client/src/Components/Search/SearchView.tsx (hunk 1 above)
Comment (ds161833) [reply Client/src/Components/Search/SearchView.tsx:21]: so the SelectionChangeParams is a type from DataGrid material component
when on the search view new rows are being selected it is reflected in the selection. selection.rowIds will contain the ids of the rows that have been selected. the bug was caused by a null exception cause selection was null and it was trying to access rowIds.

**Pull request merged**